Neos.Flow/Classes/ObjectManagement/DependencyInjection/ProxyClassBuilder.php

```diff
@@ -688,4 +670,15 @@ protected function compileStaticMethods($className, ProxyClass $proxyClass)
             $compiledMethod->setMethodBody('return ' . $compiledResult . ';');
         }
     }
+
+    private function skipBuildForClass(string $className, string $objectName): bool
+    {
+        if ($className === '' || $this->compiler->hasCacheEntryForClass($className) === true) {
+            return true;
+        }
+        if ($objectName !== $className || $this->reflectionService->isClassAbstract($className)) {
+            return true;
+        }
+        return false;
```

Comment:
This could be simpified like "in the other place" I saw this in this PR…

Reply:
Fair enough … I'll polish this during the next ProxyClassBuilder refactoring.